src/main/java/emu/grasscutter/scripts/ScriptLib.java

```diff
@@ -836,7 +849,7 @@ public int AddBlossomScheduleProgressByGroupId(int groupId){
         return 0;
     }
     public int RefreshHuntingClueGroup(){
-        logger.warn("[LUA] Call unimplemented RefreshHuntingClueGroup");
+        //logger.warn("[LUA] Call unimplemented RefreshHuntingClueGroup"); //TODO: Much many calls o this garbages the log
```

Comment:
Is this still called that often with dynamic groups only on demand?

Reply:
Probably not, i missed to remove that in my last debuggin process